package ex02_exception;

import java.util.Scanner;

public class TryCatchEx3 {
	public static void main(String[] args) {
		//정수 : 5
		//결과 : 5
		
		//정수 : aaa
		//aaa(은)는 정수가 아닙니다
		Scanner sc = new Scanner(System.in);
		
		System.out.print("정수 : ");
		
		String str = "";
		
		try {
			
			str = sc.next();
			int n = Integer.parseInt(str);
			System.out.print("결과 : " + str);
			
		}catch (Exception e) {
			System.out.print(str + "은(는) 정수가 아닙니다");
		}
		
	}//main
}
